envs/environment.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
 - pip
 - mamba   # esp for windows build
 
-- pypsa>=0.24, <0.25
+- pypsa>=0.25, <0.28
```

Comment:
Can we increase the version further?

Reply:
Yeas, I have been thinking about testing for `<0.29`. The main concern is that `v0.29.0` drops `n.lopf( )` which I'm not quite sure is really a great idea. What is your feeling about that?

Reply:
Interesting; probably worth updating till <0.29 maybe?
>=0.29 can be tested in the linopy PR with linopy; that PR should fix that too.
What do you think?

Reply:
With 0.28 testing successful :)